packages/webpack/src/utils/postcss.js

```diff
@@ -113,7 +113,7 @@ export default class PostcssConfig {
       // Map postcss plugins into instances on object mode once
       config.plugins = this.sortPlugins(config)
         .map((p) => {
-          const plugin = require(p)
+          const plugin = this.nuxt.resolver.requireModule(this.nuxt.resolver.resolveModule(p))
```

Comment:
What's the purpose of require twice ?

Reply:
It's not required twice.

`resolveModule` will get you the correct path (for the project), eg. `/path/to/project/node_modules/tailwindcss/lib/index.js`

`requireModule` will then require the resolved path.

If you mean that we could switch out `requireModule` for `require`, yes, that'd be possible

Reply:
I mean that it seemed `requireModule` will resolve the path as well 😃

Reply:
I agree. requireModule internally does it itself

Reply:
Ohh! Good point 👍